tests/gui/test_main.py

```diff
@@ -10,6 +12,18 @@
 app = QApplication([])
 
 
+class WindowTest(unittest.TestCase):
+    def setUp(self):
+        self._clipboard = QApplication.clipboard()
+        self.window = Window(self._clipboard)
+
+    def test_clear_clipboard(self):
+        self._clipboard.setText("The clipboard is not empty")
+
+        self.window.clear_clipboard()
+        assert self.window.clipboard.text() == ""
```

Comment:
```suggestion
        assert self._clipboard.text() == ""
```

Oops, this should be the assertion. The fact that the window hold a reference to to clipboard is an implementation detail. I'll fix it tomorrow.